docs/debug/3_api_debug_setup.ipynb

```diff
@@ -0,0 +1,94 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setup\n",
+    "\n",
+    "Precision debug tools for the Transformer Engine use [Nvidia-DL-Framework-Inspect](https://github.com/NVIDIA/nvidia-dlfw-inspect) package from NVIDIA. Here, we provide brief information on how to use it with TE.\n",
+    "\n",
+    "### initialize()\n",
```

Comment:
This part looks to be an API explanation - should it not be handled by API docs?